src/main/java/cubicchunks/world/provider/VanillaCubicProvider.java

```diff
@@ -61,41 +61,42 @@ public VanillaCubicProvider(ICubicWorld world, WorldProvider provider, @Nullable
 
 		boolean useProvider = false;
 
-		if(!worldObj.isRemote)
```

Comment:
I think I accidentally pushed that...

That was supposed to be only to get it working in debug mode.